src/main/java/org/benetech/servicenet/matching/counter/UrlSimilarityCounter.java

```diff
@@ -26,36 +22,19 @@ public float countSimilarityRatio(String url1, String url2, MatchingContext cont
         if (areNormalizedAndUpperCasedDifferent(url1, url2)) {
             return NO_MATCH_RATIO;
         }
-
         if (areNormalizedDifferent(url1, url2)) {
             return uppercasedWeight;
         }
 
         return COMPLETE_MATCH_RATIO;
     }
 
-    private String normalize(String url) {
-        String result = url;
-        if (result.toUpperCase(Locale.ROOT).startsWith(HTTP)) {
-            result = result.substring(HTTP.length());
-        }
-        if (result.toUpperCase(Locale.ROOT).startsWith(HTTPS)) {
-            result = result.substring(HTTPS.length());
-        }
-        if (result.toUpperCase(Locale.ROOT).startsWith(WWW)) {
-            result = result.substring(WWW.length());
-        }
-        if (result.toUpperCase(Locale.ROOT).endsWith(TRAILING)) {
-            result = result.substring(0, result.length() - 1);
-        }
-        return result;
-    }
-
     private boolean areNormalizedAndUpperCasedDifferent(String url1, String url2) {
-        return !normalize(url1).toUpperCase(Locale.ROOT).equals(normalize(url2).toUpperCase(Locale.ROOT));
+        return !UrlNormalizationUtils.normalize(url1).toUpperCase(Locale.ROOT).
```

Comment:
The dot should be in the new line,
run the tests "./mvnw test", the checkstyle will fail